File: fixtures/scale/typescript/small/theme.ts
export interface ThemeChars {
  corner: string;
  edge: string;
  side: string;
}

const THEMES: Record<string, ThemeChars> = {
  plain: { corner: '+', edge: '-', side: '|' },
  star: { corner: '*', edge: '*', side: '*' },
  dot: { corner: '.', edge: '.', side: ':' },
};

export class Theme {
  static chars(name: string): ThemeChars {
    return THEMES[name] || THEMES.plain;
  }
}

File: fixtures/scale/python/small/theme.py
THEMES = {
    "plain": {"corner": "+", "edge": "-", "side": "|"},
    "star": {"corner": "*", "edge": "*", "side": "*"},
    "dot": {"corner": ".", "edge": ".", "side": ":"},
}


def theme_chars(name):
    return THEMES.get(name, THEMES["plain"])
